src/tuple_gen.h

```diff
@@ -886,6 +928,16 @@ struct CTupleGenYamlInfo {
         }
         return 0;
     }
+    
+    void set_server_track_ports(uint8_t svr_pool_id, uint8_t plugin_id) {
+        if (plugin_id!=0) {
+            if (m_server_pool[svr_pool_id].m_is_bundling==false) {
```

Comment:
If user configures plugin_id and not configures track_ports, I'll force to enable track_ports. 
I'm thinking do we need such logic? Not every plugin needs to track server port, right? 

Or I can simply print out a warning without force to enable track_ports?